tests/sentry/replays/test_project_replay_summarize_breadcrumbs.py

```diff
@@ -145,5 +236,5 @@ def _faker():
             ).encode()
         )
 
-    result = get_request_data(_faker())
+    result = get_request_data(_faker(), error_events=[])
```

Comment:
It's valuable I think to test this with at least one error so you can see it interleave.  And optionally one after the stream so its appended to the end of the logs.